### PR TITLE
Menus more semantic and accessible

### PR DESCRIPTION
**Pull request by Valioesi**
There were some issues with the menu in header and footer. Not very semantic. No `<header>`, no `<ul>`, etc. And also the submenus were not tabable.

### CODE REVIEW

File: layout/Footer/FooterMenu/index.tsx [hunk 1]
```diff
@@ -10,16 +10,16 @@ type FooterMenuProps = {
 export const FooterMenu = ({ footerMenu }: FooterMenuProps): ReactElement => {
   const entries = footerMenu.filter(e => 'slug' in e) as MenuEntry[];
   return (
-    <div className={s.footerMenuRow}>
+    <ul className={s.footerMenuRow}>
```
Comment (schaerta) [inline layout/Footer/FooterMenu/index.tsx:13]: Why not <nav>?

Comment (schaerta) [reply layout/Footer/FooterMenu/index.tsx:13]: ´<nav>´

Comment (schaerta) [inline layout/Footer/FooterMenu/index.tsx:13]: Why not `<nav>`?

Comment (schaerta) [reply layout/Footer/FooterMenu/index.tsx:13]: Ah, this is inside a nav-tag, right? Is it really better to use `<ul>` for menus? I didn't know that.

Comment (Valioesi) [reply layout/Footer/FooterMenu/index.tsx:13]: Nope, in the footer it's not inside nav. Yeah, I thought about using nav and actually researched a bit. According to specification: 

> Not all groups of links on a page need to be in a [nav](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) element — the element is primarily intended for sections that consist of major navigation blocks. In particular, it is common for [footer](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element)s to have a short list of links to various pages of a site, such as the terms of service, the home page, and a copyright page. The footer element alone is sufficient for such cases; while a [nav](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) element can be used in such cases, it is usually unnecessary.

And ul just makes sense semantically for lists. In menus it also seems common practice to me. I rarely see a nav without ul.

Comment (Valioesi) [reply layout/Footer/FooterMenu/index.tsx:13]: There's also stackoverflow threads about the ul thing: https://stackoverflow.com/questions/5544885/should-i-use-uls-and-lis-inside-my-navs

Comment (Valioesi) [reply layout/Footer/FooterMenu/index.tsx:13]: But it's probably not a huge deal to use divs instead

Comment (schaerta) [reply layout/Footer/FooterMenu/index.tsx:13]: Interesting! Thanks, I learned something new today :)

**Review by schaerta — APPROVED** [review]

**Pull request merged**